explanation/intro-to/security.md

```diff
@@ -41,27 +42,26 @@ anyone using Ubuntu.
 
 All of the Ubuntu Pro features can be managed on the command line via the
 [Ubuntu Pro Client](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/)
-utility, which also has an API for easier automation.
+utility, which has an API that provides easier automation to users.
 
 ### Vulnerability management
 
-In a standard Ubuntu LTS release, security support is provided for packages in
-the [Main repository](https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/explanation/archive/#components)
-for 5 years. With Ubuntu Pro, this is expanded to 10 years, and also includes
-patching for medium, high and critival severity
+In a standard Ubuntu LTS release, Expanded Security Management security support is provided 
```

Comment:
```suggestion
In a standard Ubuntu LTS release, security support is provided 
```

Reply:
Expanded Security Management doesn't exist as a concept - with Ubuntu Pro we have to be *very* careful about wording to ensure we're not promising things that won't be delivered (and we need to be especially careful about the terminology we use).